Makefile

```diff
@@ -9,7 +9,7 @@
 SHELL=/bin/bash -o pipefail
 
 PROJECT_NAME=pravega-operator
-REPO=pravega/$(PROJECT_NAME)
+REPO=tristan1900/$(PROJECT_NAME)
```

Comment:
We should not be using private repos here

Reply:
Sure

Reply:
As discussed, we can continue using this till permissions issue for "pravega" repo is resolved.